src/briefcase/platforms/linux/system.py

```diff
@@ -760,15 +760,39 @@ def build_app(self, app: AppConfig, **kwargs):
                 )
 
         with self.console.wait_bar("Installing changelog..."):
-            changelog = self.base_path / "CHANGELOG"
-            if changelog.is_file():
-                with changelog.open(encoding="utf-8") as infile:
-                    outfile = gzip.GzipFile(
-                        doc_folder / "changelog.gz", mode="wb", mtime=0
-                    )
-                    outfile.write(infile.read().encode("utf-8"))
-                    outfile.close()
-            else:
+            changelog_name_formats = [
+                "CHANGELOG",
+                "HISTORY",
+                "NEWS",
+                "RELEASES",
+                "CHANGELOG.md",
+                "CHANGELOG.rst",
+                "CHANGELOG.txt",
+                "HISTORY.md",
+                "HISTORY.rst",
+                "HISTORY.txt",
+                "NEWS.md",
+                "NEWS.rst",
+                "NEWS.txt",
+                "RELEASES.md",
+                "RELEASES.rst",
+                "RELEASES.txt",
+            ]
```

Comment:
Two notes: 

1. When you're enumerating business logic that includes a complex list like this for the second time, you should be considering whether it should be pulled out as a utility method (or at the very least, a constant). A `find_changelog_filename()` method that returns the match (or None/exception if doesn't exist) would seem appropriate here.
2. You can generate this list in a much less error prone way with a list comprehension: `[f"{name}{ext}" for name in ["CHANGELOG", "HISTORY"] for extension in ["", ".md"]]`